homeassistant/helpers/config_validation.py

```diff
@@ -219,6 +221,17 @@ def validate(obj):
     return validate
 
 
+def validate_values(valid_values):
```

Comment:
Ohh, thanks. Just learning voluptuous.

I am not able to use vok.In
Have tried: { vol.In(SENSOR_TYPES.keys())},
but get the following error: voluptuous.SchemaError: unsupported schema data type 'set'

Do you have any suggestions?